vm/rust/Cargo.toml

```diff
@@ -8,8 +8,11 @@ edition = "2021"
 [dependencies]
 serde = "1.0.171"
 serde_json = { version = "1.0.96", features = ["raw_value"] }
-blockifier = { git = "https://github.com/NethermindEth/blockifier", branch = "native2.6.4" }
-# blockifier = { path = "../../../blockifier-neth/crates/blockifier/" }
+# blockifier = { git = "https://github.com/NethermindEth/blockifier", branch = "native2.6.4" }
```

Comment:
Before merging, please set the Blockifier dependency back to track the Github repo